app/components/work_versions/version_navigation_component.rb

```diff
@@ -37,7 +38,13 @@ def path_for(version)
       end
     end
 
-    def classes_for(version)
+    def list_classes_for(version)
+      classes = []
+      classes << 'active' if current?(version)
+      classes.join(' ')
```

Comment:
I'm not seeing any other classes, so the return for this function would either be `nil` or `'active'`. Is that right? If so, you can just rewrite it as:

``` ruby
def list_classes_for(version)
  return unless current(version)

  'active'
end

Reply:
@awead - Good call. The original version of this did the same, but also had one class. I thought that was kinda strange but I guess in the moment "preserve original style" won out over "refactor this" haha. 🙂

Reply:
No, I think "preserve original style" is the best first step. I often forget that and jump straight to refactoring, when really it's better to have the refactoring emerge from the keeping-things-looking-the-same.